01-wasmcloud_basics-pathway.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "wasmcloud basics",
+  "description": "washcloud Beginner Tutorials",
+  "courses": [
+    {
+      "title": "wash Introduction",
+      "description": "Introduction to wasmcloud REPL",
+      "course_id": "wash_intro"
+    },
+    {
+      "title": "wasmcloud Introduction",
```

Comment:
I actually had them ordered this way at one point and then overrode the change.  I agree with this.  I will do all the top level edits in a different PR so that I dont have to rebase the branches.